webview/src/plots/components/App.test.tsx

```diff
@@ -1416,7 +1470,9 @@ describe('App', () => {
       template: complexTemplatePlotsFixture
     })
 
-    const plot = within(screen.getAllByTestId(/^plot_/)[0]).getByRole('button')
+    const plot = within(screen.getAllByTestId(/^plot_/)[0]).getByTestId(
+      'zoomable-plot'
```

Comment:
There are now two possible buttons (plot itself and our actions button). Added an id to differentiate.

Reply:
[C] could have added an `aria-label` and used that instead.